{{cookiecutter.project_slug}}/README.md

```diff
@@ -0,0 +1,12 @@
+# {{ cookiecutter.project_slug }}
+
+{{ cookiecutter.package_description }}
+
+### Setup
+
+```bash
+pip install {{ cookiecutter.project_slug }}
```

Comment:
I think this assumes the package will be on pypi, which is not the case with private packages.

Reply:
Yes, because that would be the default case. By default what we build is open source unless it contains a USP or IP.